integration/js/app/video_test/video_test.html

```diff
@@ -62,6 +62,76 @@ <h2>Join a meeting</h2>
           Unbind Video Element
         </button>
       </form>
+
+       
+      <form id="add-video-tile-form">
+        <button id="add-video-tile-btn" type="submit">
+          addVideoTile
+        </button>
+        <label for="add-video-tile-label">Tile Id: </label>
+        <span id="video-tile-id"></span>
+      </form>
+
+      <form id="has-started-local-video-tile-form">
+        <button id="has-started-local-video-tile-btn" type="submit">
+          hasStartedLocalVideoTile
+        </button>
+        <label for="has-started-local-video-tile-label">Value: </label>
+        <span id="has-started-local-video-tile-boolean"></span>
+      </form>
+
+      <form id="get-local-video-tile-form">
+        <button id="get-local-video-tile-btn" type="submit">
+          getLocalVideoTile
+        </button>
+        <label for="local-video-tile-label">Tile Id: </label>
+        <span id="local-video-tile-id"></span>
+      </form>
+
+      <form id="get-all-video-tiles-form">
+        <button id="get-all-video-tiles-btn" type="submit">
+          getAllVideoTiles
+        </button>
+        <label for="total-video-tiles-label">Total Tiles Count: </label>
```

Comment:
Maybe it would be useful to return the actual VideoTile Objects here instead of just the count.

Reply:
You mean compare the Two objects with the expected two objects to verify the response? Not sure if it is necessary but just compare the count could probably be enough since in the corresponding step in the videoTest, we only compare the number on roster to verify both attendees have joined the meeting.

Reply:
I think the main reason is for validation purposes. The VideoTile is an object full of metadata that would make it easier to see what is happening to those videoTile objects behind the scenes as the video tile states are changing